pkg/admissioncontroller/apis/config/validation/admissionControllerConfiguration.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
fix file name, ref https://stackoverflow.com/questions/25161774/what-are-conventions-for-filenames-in-go